server/events/tests/test_models.py

```diff
@@ -0,0 +1,36 @@
+from datetime import timedelta
+from django.utils import timezone
+from django.contrib.auth import models as auth_models
+from .shared_case import SharedTestCase
```

Comment:
The location of this file can change, so it's better to use imports like `from events.tests import shared_case`.

Reply:
fixed